src/crc32cer.erl

```diff
@@ -24,24 +24,15 @@ nif(_Acc, _IoData) ->
 
 -spec so_path() -> string().
 so_path() ->
-  filename:join([get_nif_bin_dir(), "crc32cer"]).
-
-get_nif_bin_dir() ->
-  {ok, Cwd} = file:get_cwd(),
-  get_nif_bin_dir(
-    [ code:priv_dir(?APPLICATION)
-    , filename:join([Cwd, "..", "priv"])
-    , filename:join(Cwd, "priv")
-    , os:getenv("NIF_BIN_DIR")
-    ]).
-
-get_nif_bin_dir([]) -> erlang:error(crc32cer_nif_not_found);
-get_nif_bin_dir([false | Rest]) -> get_nif_bin_dir(Rest);
-get_nif_bin_dir([Dir | Rest]) ->
-  case filelib:wildcard(filename:join([Dir, "crc32cer.*"])) of
-    [] -> get_nif_bin_dir(Rest);
-    [_ | _] -> Dir
-  end.
+  filename:join(case code:priv_dir(?MODULE) of
```

Comment:
not sure why we need this
@gilbertwong96

Reply:
or why the original code does not work?

Reply:
Sorry, I totally forget about the meaning of these commits. It’s should be something wrong happened on windows and I hacked them with these codes.

Reply:
I would prefer to have this reverted, after merge
could you help to verify if it works in windows ? @gilbertwong96